WalletWasabi/Helpers/Constants.cs

```diff
@@ -33,6 +33,14 @@ public static class Constants
 	public const int P2trInputVirtualSize = 58;
 	public const int P2trOutputVirtualSize = 43;
 
+	public const int P2pkhInputVirtualSize = 148;
+	public const int P2pkhOutputVirtualSize = 34;
+	public const int P2wshInputVirtualSize = 105; // we assume a 2-of-n multisig
+	public const int P2wshOutputVirtualSize = 32;
+	public const int P2shInputVirtualSize = 297; // we assume a 2-of-n multisig
+	public const int P2shOutputVirtualSize = 32;
```

Comment:
There is no bigger p2(w)sh outputs because the `h` in p2(w)sh means `hash` and that is a 32 bytes always. I mean, you don't put the script in the output but the hash of it instead.

The script size matters for the input script (scriptSig) and that's not really our problem because wasabi doesn't support those script types. However, we assume a size because we use that size value to estimate whether the output that the wallet wants to add is economical or not. So, in case the script is much bigger than what we assumed the output will be expensive to spend.